engine/proto/master.proto

```diff
@@ -121,11 +114,10 @@ message Selector {
 
 message HeartbeatRequest {
     string executor_id = 1;
-    int32 resource_usage = 2;
-    int32 status = 3;
+    int32 status = 2;
```

Comment:
Should the status field be removed?

Reply:
+1

Reply:
Better to remove it, it is not used now.